test-runner/src/main/scala/com/typesafe/spark/test/mesos/framework/runners/MesosIntegrationTestRunner.scala

```diff
@@ -4,14 +4,20 @@ import com.typesafe.config.ConfigFactory
 import com.typesafe.spark.test.mesos.framework.runners.Utils._
 
 object MesosIntegrationTestRunner {
-
-
   def main(args: Array[String]): Unit = {
-
     implicit val config = ConfigFactory.load()
     printMsg(config.toString)
 
-    val failures: Int = ClientModeRunner.run(args) + ClusterModeRunner.run(args) + MultiClusterModeRunner.run(args)
+    // TODO(tnachen): Add support with a args parsing library to avoid
+    // parsing args based on positions and support flags.
+    val withZk = args.exists(a => a.equals("--with-zk"))
```

Comment:
I see, I was thinking we wanted to have a opt-in even when zk.uri is set.
I think having it being configured is fine with me, I'll just reuse the same check then. I don't think we need a helper just to check a config for now.